Edit: edited hotel model

ecila7290 opened the pull request:
- models.py 수정사항 반영
 - 필드명 수정
 - 필드 속성에 맞는 django Field 사용
 - HotelInformation 테이블과 Infolocation의 1:N관계를 고려한 필드 삭제
 - HotelInformation테이블의 db_table을 클래스명에 맞춰 수정

>wecode PR message template 및 체크 리스트 입니다. 
아래 사항들을 전부 작성/체크 하시고 PR 해주세요!

## 수정 사항 간략한 한줄 요약
(여기에 수정 사항에 대한 간략한 한줄 요약/제목 작성해 주세요.)
- 로그인 버튼 활성화 기능 구현(예시)
		 
## 수정 사항들 자세한 내용
(여기에 수정 사항에 대한 자세한 내용을 작성해 주세요.)
- id '@' 포함, pw 5 글자 이상 입력 시 버튼 활성화(예시)

## 기타 질문 및 특이 사항
(궁금한 사항들, 하면서 느낀 점들 공유해주실 것이 있으면 자유롭게 작성해 주세요.)
- 함수 로직이 너무 긴 것 같습니다. 좀 더 효율적인 방법이 없을지 궁금합니다.(예시) 

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [ ] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [ ] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [ ] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ ] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

ecila7290 committed:
Add: added hotel model
 - 호텔 앱 model 추가
 - model migrate

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    english_name       = models.CharField(max_length=200)
+    address            = models.CharField(max_length=200)
+    introduction       = models.CharField(max_length=200)
+    thumbnail          = models.URLField(max_length=1000)
```
```suggestion
    thumbnail_url          = models.URLField(max_length=1000)
```

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    min_people         = models.IntegerField()
+    max_people         = models.IntegerField()
+    room_count         = models.IntegerField()
+    checkin_time       = models.CharField(max_length=20)
+    checkout_time      = models.CharField(max_length=20)
```
시간을 관리할 때에는 Datetime을 입력할 수 있는 필드를 고려해보세요 :)

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    room_count         = models.IntegerField()
+    checkin_time       = models.CharField(max_length=20)
+    checkout_time      = models.CharField(max_length=20)
+    x                  = models.DecimalField(max_digits=12, decimal_places=7)
+    y                  = models.DecimalField(max_digits=12, decimal_places=7)
```
```suggestion
    logitude                  = models.DecimalField(max_digits=12, decimal_places=7)
    latitude                  = models.DecimalField(max_digits=12, decimal_places=7)
```
다른 개발자 혹은 사용자가 보았을 때 어떤 의미인지 알기 쉽도록 지정해주세요 :)

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    checkout_time      = models.CharField(max_length=20)
+    x                  = models.DecimalField(max_digits=12, decimal_places=7)
+    y                  = models.DecimalField(max_digits=12, decimal_places=7)
+    tel_number         = models.CharField(max_length=200)
```
```suggestion
    phone_number         = models.CharField(max_length=200)
```

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    x                  = models.DecimalField(max_digits=12, decimal_places=7)
+    y                  = models.DecimalField(max_digits=12, decimal_places=7)
+    tel_number         = models.CharField(max_length=200)
+    email              = models.CharField(max_length=200)
```
이메일을 관리하기 위한 email 필드가 있으니 활용해보세요 !

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    introduction  = models.TextField()
+    min_people    = models.IntegerField()
+    max_people    = models.IntegerField()
+    area          = models.IntegerField()
+    price         = models.IntegerField()
+    name          = models.CharField(max_length=200)
+    price_weekday = models.IntegerField()
+    price_weekend = models.IntegerField()
+    price_peak    = models.IntegerField()
```
면적, 가격과 같은 요소들은 보통 소수점 자리 둘째자리, 혹은 넷째자리까지 관리해줍니다 :) 대부분의 통화들이 소수점까지 사용해서 관리하는 경우가 많기 때문입니다

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    place1_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place1_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place2_name   = models.CharField(max_length=40)
+    place2_detail = models.CharField(max_length=200)
+    place2_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place2_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place3_name   = models.CharField(max_length=40)
+    place3_detail = models.CharField(max_length=200)
+    place3_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place3_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place4_name   = models.CharField(max_length=40)
+    place4_detail = models.CharField(max_length=200)
+    place4_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place4_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place5_name   = models.CharField(max_length=40)
+    place5_detail = models.CharField(max_length=200)
+    place5_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place5_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place6_name   = models.CharField(max_length=40)
+    place6_detail = models.CharField(max_length=200)
+    place6_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place6_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place7_name   = models.CharField(max_length=40)
+    place7_detail = models.CharField(max_length=200)
+    place7_x      = models.DecimalField(max_digits=12, decimal_places=7)
+    place7_y      = models.DecimalField(max_digits=12, decimal_places=7)
+
+    place8_name   = models.CharField(max_length=40)
```
OneToMany 관계로 설정하셨다면, 모든 음식점의 정보를 1번부터 8번까지 지정하는 것은 1:N 관계를 최대한 활용하지 못한 것이 됩니다. 이름과 상세 정보, 위/경도 정보로 통일해주세요

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    class Meta: 
+        db_table = 'room_facilities'
+
+class MoreInfo(models.Model): 
```
`HotelInformation` 등 알기 쉬운 이름으로 설정하시는 것이 좋습니다.

soheon-lee commented on hotel/models.py:
```diff
@@ -0,0 +1,207 @@
+    description       = models.CharField(max_length=200)
+    detail_top        = models.TextField()
+    detail_content    = models.TextField()
+    detail_sign       = models.CharField(max_length=20)
```
```suggestion
    detail_signature       = models.CharField(max_length=20)
```

ecila7290 committed:
Edit: edited model
 - models.py 수정사항 반영
 - 필드명 수정
 - 필드 속성에 맞는 django Field 사용
 - HotelInformation 테이블과 Infolocation의 1:N관계를 고려한 필드명 삭제

ecila7290 committed:
Edit: edited table name
 - HotelInformation테이블의 db_table을 클래스명에 맞춰 수정

soheon-lee commented:
LGTM
